src/parser/util.js

```diff
@@ -86,3 +87,25 @@ pp.unexpected = function (pos, messageOrType = "Unexpected token") {
   }
   this.raise(pos != null ? pos : this.state.start, messageOrType);
 };
+
+pp.expectPlugin = function (name) {
+  if (!this.hasPlugin(name)) {
+    const pluginInfo = plugins[name];
+    const pointers = [];
+    if (pluginInfo) {
+      if (pluginInfo.babelSyntaxPlugin) {
+        pointers.push(`babel-plugin-${pluginInfo.babelSyntaxPlugin}`);
+      }
+      if (pluginInfo.babelTransformPlugins) {
+        pointers.push(...pluginInfo.babelTransformPlugins.map(
+          (shortName) => `babel-plugin-${shortName}`
+        ));
+      }
+    }
+    const leadingPointersOr = pointers.length ? ` ${pointers.join(", ")} or ` : " ";
```

Comment:
Yeah that was my thoughts dono if it was in the notes. Babylon would just throw and babel-core would pick that up and throw the real error message since you might be using babylon for something else (prettier, babel-eslint) that is unrelated to using a plugin